src/pages/AddScholarship.jsx

```diff
@@ -19,7 +19,18 @@ function AddScholarship() {
       max: 0,
     },
     website: '',
+    organization: '',
+    tags: [],
     author: { id: user?.uid, email: user?.email },
+    requirements: {
+      gpa: null,
```

Comment:
Careful with null-safety. 0 is a safer value and it encapsulates well that any GPA is fine.